### DISCUSSION TITLE
Lightning menu name + trigger?

### DISCUSSION THREAD

**Discussion by klausbreyer** [zed-industries/zed#8262] [Feature Requests]
I tried to find it in the docs, but it is hard to look for something where you don't know the name.. 

So what is the name of this lighting menu? And how can I trigger it with a keybinding? 

<img width="328" alt="grafik" src="https://github.com/zed-industries/zed/assets/32771/bea51d23-2715-410a-9eee-87e73e59ec13">

Context: I really need an quick and easy way to organize imports and remove unused imports, etc. 
([Best case: on save. But it seems this is not there, yet](https://github.com/zed-industries/zed/issues/4840)). Until then I hope to have a shortcut for those lightning menu actions.

**Comment by mrnugget** [comment]
It's called code actions. You can trigger them with <kbd>cmd-.</kbd> (Spec: https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_codeAction)

It sounds like what you want is `code_actions_on_format`, which was added in https://github.com/zed-industries/zed/pull/7860 and landed in preview release here: https://zed.dev/releases/preview/0.124.0

I'll reply in the other ticket too!

**Comment by klausbreyer** [comment]
oh cool, thank you very much!
